sourmash/signature_json.py

```diff
@@ -253,32 +245,62 @@ def save_signatures_json(siglist, fp=None, indent=None, sort_keys=True):
         x.append(sketch)
         top_records[k] = x
 
-    if not top_records:
-        return ""
+    return top_records
 
+
+def add_meta_save(top_records):
+    """ Convert one signatures into a JSON dict
+    - siglist: sequence of SourmashSignature objects
+    - index: index of siglist to save
+    """
+    notify("in add_meta_save", end="\r")
+    from .signature import SIGNATURE_VERSION
     records = []
     for (name, filename), sketches in top_records.items():
         record = {}
         if name:
-            record['name'] = name
+            record["name"] = name
         if filename:
-            record['filename'] = filename
-        record['signatures'] = sketches
-
-        record['version'] = SIGNATURE_VERSION
-        record['class'] = 'sourmash_signature'
-        record['hash_function'] = '0.murmur64'
-        record['license'] = 'CC0'
-        record['email'] = ''
-
+            record["filename"] = filename
```

Comment:
same comment here - these changes don't seem to be significant, can you revert back to single quotes so we can see what was actually changed, if anything?